packages/lighthouse/src/index.ts

```diff
@@ -22,11 +22,14 @@ const lhConfig = ({ urls, server, assertions = {} }: Params) => {
       assert: {
         preset: 'lighthouse:no-pwa',
         assertions: {
-          // Final Ligthouse score Budgets
+          // Final Lighthouse score Budgets
```

Comment:
I'll move this to another PR.

Page is blocked from indexing when is in preview. So I think we can set it as a warn when not in production.